bin/build

```diff
@@ -4,26 +4,31 @@ set -e
 CURRENT_DIR=$(pwd)
 
 if [[ ! $INPUT_DIR ]] || [[ ! $OUTPUT_DIR ]]; then
-  echo "Input/Output directory variables must be set.\n"
+  printf "Input/Output directory variables must be set.\n"
   exit 1
 fi
 
 mkdir -p $INPUT_DIR
 mkdir -p $OUTPUT_DIR
 
 if [[ ! -d $INPUT_DIR ]] || [[ ! -d $OUTPUT_DIR ]]; then
-  echo "Input/Output directories must exist.\n"
+  printf "Input/Output directories must exist.\n"
   exit 1
 fi
 
 NVM_DIR=$HOME/.nvm
 source /opt/nvm/nvm.sh
 
+# Copy Meteor distribution to $HOME
+printf "Initializing Meteor environment.\n"
+cp -R /opt/home/.meteor $HOME
+
 # Demeteorizer flags
 # TODO Test for demeteorizer flags
 # --json || --debug
 
 cd $INPUT_DIR
+printf "Using $(meteor --version).\n"
```

Comment:
This is not output on all builds. I'm testing with the Meteor Leaderboard example app with `Meteor@1.2.1`